config/services.yml

```diff
@@ -68,6 +68,8 @@ services:
     class: Symfony\Component\EventDispatcher\EventDispatcher
     public: true
 
+  Symfony\Component\Console\Output\OutputInterface:
+    alias: Symfony\Component\Console\Output\ConsoleOutput
```

Comment:
just allows us to depend on the interface instead of the implementation and still benefit from the dreaded auto-wiring.